packages/lisk-transactions/test/utils/validation/validate_transaction.js

```diff
@@ -54,7 +54,7 @@ describe('validateTransaction', () => {
 		describe('when the transaction contains invalid data in merged schema', () => {
 			const invalidTransaction = {
 				type: 0,
-				amount: '0',
+				amount: '1',
```

Comment:
Probably it's better to add `transferAmount` test here too, because it's also invalid merged schema pattern